src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1952,63 +1952,44 @@ because xbuild doesn't support framework reference assemblies.
   <!-- Create our intermediate directory for assemblies -->
   <MakeDir Directories="$(MonoAndroidIntermediateAssemblyDir)" Condition=" !Exists ('$(MonoAndroidIntermediateAssemblyDir)') " />
 </Target>
+
+<Target Name="_CollectPlatformJarFilenames">
+	<ItemGroup>
```

Comment:
This should be indented with spaces, to fit in with the surrounding code.  (Plus spaces are generally preferable in our XML files.)